src/components/Vuetify/mixins/breakpoint.ts

```diff
@@ -1,90 +1,62 @@
 import Vue from 'vue'
 import { VuetifyBreakpoint } from 'types'
 
+const breakpoints = {
+  xs: 600,
+  sm: 960,
+  md: 1280 - 16,
+  lg: 1920 - 16,
+  xl: Infinity
+}
+const keys = Object.keys(breakpoints) as (keyof typeof breakpoints)[]
+const values = Object.values(breakpoints)
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
+type Static = Omit<VuetifyBreakpoint, 'width' | 'height'>
+const breakpointExplainations: Static[] = keys.map((name, rowIndex) => {
+  const result: Record<
+  string,
+  keyof Omit<VuetifyBreakpoint, 'width' | 'height'> | boolean
+  > = { name }
+  keys.forEach((key: keyof typeof breakpoints, i) => {
+    result[key] = result[`${key}Only`] = i === rowIndex
+    result[`${key}AndDown`] = i >= rowIndex
+    result[`${key}AndUp`] = i <= rowIndex
+  })
+  return result as Static
```

Comment:
No idea what you've done with the types here, `result` should be defined as `Record<string, Static | boolean>` but then you're asserting it to `Static` instead.

Reply:
Nuh-uh, that's the `return` of the function being mapped.
It returns a single row of the lookup table, containing all the static results (hence the name) for the viewport range. This includes one string, `name`, and the rest booleans.
A dense integer-indexed array of these, `Static[]` is the product of the whole mapping and the lookup table we want.
Finally, the reason I had to assert is this object is created in a loop, furthermore, through string concatenation of keys. Hence, it doesn't know of any specific key except `name`, and manages to lose even that.

To be completely honest, it actually produces more than a `Static`, by including the nonsense of `xsAndDown` and `xlAndUp` and always-`true` of `xlAndDown` and `xsAndUp`, but I am not sure whether putting a condition to not create them is necessary. Please advise.

Would be this:
```ts
const max = keys.length - 1
keys.forEach((key, i) => {
  result[key] = result[`${key}Only`] = i === rowIndex
  if (i === 0 || i === max) return
  result[`${key}AndDown`] = i >= rowIndex
  result[`${key}AndUp`] = i <= rowIndex
})
```